src/patternfly/components/DualListSelector/dual-list-selector-controls-item.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-dual-list-selector__controls-item{{#if dual-list-selector-controls-item--modifier}} {{dual-list-selector-controls-item--modifier}}{{/if}}"
```

Comment:
`item` is typically what we call a generic child of some parent like this, and helps differentiate between `__controls` and `__control` if we went with that naming since they're so similar. We use `control`/`controls` in other places - controls is what we used in code editor https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/CodeEditor/code-editor-controls.hbs